README.md

```diff
@@ -223,3 +223,43 @@ use stage 1.
 ```
 ./stage2/bin/zig build --build-file ../build.zig install -Drelease-fast
 ```
+
+## Developing Zig
+
+### Standard Library
+
+First, build the Stage 1 compiler as described in [the Building section](#building).
```

Comment:
Great idea! I think this will be helpful for people who want to start contributing.

One note: I think the example at the bottom might actually be more confusing than helpful. As a newcomer it can be really difficult to tell what commands are meaningful or required, even when you write it out. Anything beyond the bare minimum can be overwhelming. My recommendation would be numbered steps, something like this:

```suggestion
First, build the Stage 1 compiler as described in [the Building section](#building).
To test changes to the standard library, do the following from the build directory:

1. Make your changes to the standard library files in `std`.
2. Run `make install` (on POSIX) or
`msbuild -p:Configuration=Release INSTALL.vcxproj` (on Windows).
3. Test your changes with `bin/zig test <changed file>`.

Once your changes are finished, run all the zig tests from the build directory:

```
bin/zig build --build-file ../build.zig test -Dskip-release
```
```

Reply:
(Pardon the effed-up formatting. Markdown inside Markdown doesn't really work.)

Reply:
I like the numbered list, will make that change.

The primary reason I included the example at the bottom is to communicate specifically *which* files should be modified, as one confusion I had when making my first change was whether I should change the files in `zig/std` (which has no effect unless you `make install`) or in `zig/build/lib/zig/std` (which has an immediate effect but those files aren't tracked by git). I couldn't figure out how exactly to word the instructions to make that bit clear, so I figured a more complete example would hopefully make that bit clearer.

Reply:
Oh, I see, that's a good point. If that causes confusion, maybe it be worth explicitly pointing out that distinction?

```
Note that you should make your changes to files in the `std` directory, not `build/lib/zig/std`;
the latter will not be tracked by git.
```

Anyway, details -- good change overall. :+1:

Reply:
Let me know what you think of the updated version. I moved the first item out of the numbered list since "do the following from the build directory" didn't apply to it.

Reply:
Looks fantastic @squeek502 . Moving `-Dskip-release` to a li'l footer note feels just right too.

(By the way, thanks for that -- I was looking for the flag which skips non-native builds.)